receivers/discord/config_test.go

```diff
@@ -0,0 +1,88 @@
+package discord
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/alerting/receivers"
+	testing2 "github.com/grafana/alerting/receivers/testing"
```

Comment:
nit: can this be something a bit more meaningful? e.g. `receivers_testing`

Reply:
yes. I will address it in the follow up!